swap/src/cli/api/tauri_bindings.rs

```diff
@@ -172,7 +190,22 @@ pub enum TauriSwapProgressEvent {
 #[typeshare]
 #[derive(Debug, Serialize, Clone)]
 #[typeshare]
-pub struct CliLogEmittedEvent {
+pub struct TauriLogEvent {
     /// The serialized object containing the log message and metadata.
-    pub buffer: String
+    pub buffer: String,
+}
+
+#[derive(Serialize, Clone)]
+#[typeshare]
+pub struct TauriDatabaseStateEvent {
+    #[typeshare(serialized_as = "string")]
+    swap_id: Uuid,
+}
+
+#[derive(Serialize, Clone)]
+#[typeshare]
+pub struct TauriTimelockChangeEvent {
+    #[typeshare(serialized_as = "string")]
+    swap_id: Uuid
+    // Maybe add which timelock changed? E.g. cancel, refund, punish
```

Comment:
All the data we need is inside the ExpiredTimelock object. No need for anything else.